public/app/modules/post/post.html

```diff
@@ -0,0 +1,38 @@
+<div class="container-fluid">
+	<div class="row" ng-include="'app/modules/partials/navbar.html'"></div>
+</div>
+<div class="row">
+	<div class="no-padding col-sm-12 col-md-12 col-lg-12">
+		<div id="hero" style="background-image: url({{post.image.secure_url}});"></div>
+	</div>
+</div>
+<div class="container">
+	<div class="row">
+		<div class="article-body white-background col-sm-12 col-md-12 col-lg-12">
+			<div class="row">
+				<div class="article no-padding col-sm-8 col-md-9 col-lg-9">
+					<div class="label label-text-2 black-text">
+						IDEAS
```

Comment:
@matjames007 I believe this is an artifact of the design process. We will have a discussion around what should be used here.